src/net.cpp

```diff
@@ -81,6 +81,9 @@ struct ListenSocket
 };
 }
 
+/** Services this node implementation cares about */
+static const uint64_t nRelevantServices = NODE_NETWORK;
```

Comment:
hmm, while i understand we don't want to connect to only pruned nodes outbound, i wonder that pruned nodes will get very few connections..basically the pruned node will make it's own outbound connections but get very few incoming if any...is that OK?  i'm wondering...it seems that Core's approach is that pruned nodes offer little to the network...if true then they the code and the previous commit are of value...but if not then we need a different approach.  I don't know now the answer...maybe we need a wider discussion about how we're going to handle non NETWORK nodes.